From 07dab7fe552c53e7840e34d3c8bb1cc43a921706 Mon Sep 17 00:00:00 2001
From: "aperez@igalia.com"
 <aperez@igalia.com@268f45cc-cd09-0410-ab3c-d52691b4dbfc>
Date: Wed, 12 Jul 2017 18:43:36 +0000
Subject: [PATCH] bmalloc: Failure to build when the compiler specifically
 targets ARMv8-A / defines __ARM_ARCH_8A__
 https://bugs.webkit.org/show_bug.cgi?id=174424

Reviewed by Michael Catanzaro.

* bmalloc/BPlatform.h: Also check for __ARCH_ARM_8A__ to detect ARMv8.

Signed-off-by: Adrian Perez de Castro <aperez@igalia.com>

git-svn-id: http://svn.webkit.org/repository/webkit/trunk@219416 268f45cc-cd09-0410-ab3c-d52691b4dbfc
---
 Source/bmalloc/bmalloc/BPlatform.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Source/bmalloc/bmalloc/BPlatform.h b/Source/bmalloc/bmalloc/BPlatform.h
index 8d768db63fb..400143a63fb 100644
--- a/Source/bmalloc/bmalloc/BPlatform.h
+++ b/Source/bmalloc/bmalloc/BPlatform.h
@@ -120,7 +120,8 @@
 || defined(__ARM_ARCH_7S__)
 #define BARM_ARCH_VERSION 7
 
-#elif defined(__ARM_ARCH_8__)
+#elif defined(__ARM_ARCH_8__) \
+|| defined(__ARM_ARCH_8A__)
 #define BARM_ARCH_VERSION 8
 
 /* MSVC sets _M_ARM */
-- 
2.13.3

